### PR TITLE
Added space using spacer variables on home page (#1zhjrv3)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
oie_sO54xehVcGiz

File: src/views/Home.vue [hunk 1]
```diff
@@ -158,7 +158,7 @@ export default defineComponent({
 }
 
  .internal {
-   margin-top: 24px;
+   margin: var(--spacer-sm) var( --spacer-xs) 0 var( --spacer-xs);
```
Comment (dt2patel) [inline src/views/Home.vue]: add space between individual ion items too

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/Home.vue (hunk 1 above)
Comment (azkyakhan) [reply src/views/Home.vue]: Done sir
internal -items
internal-items2

File: src/views/Home.vue [hunk 2]
```diff
@@ -157,13 +157,10 @@ export default defineComponent({
   height: 200px;
 }
 
- .internal {
-   margin-top: 24px;
```
Comment (dt2patel) [inline src/views/Home.vue:161]: We want to have some margin here so that the internal section is a little distanced from the external. Spacer base maybe

**Review by dt2patel — CHANGES_REQUESTED** [review]

File: src/views/Home.vue (hunk 2 above)
Comment (azkyakhan) [reply src/views/Home.vue:161]: Done sir

**Pull request merged**